### ISSUE TITLE
Ensure the NetBSD test suite runs OK

### ISSUE THREAD

**Issue by jmmv** [jmmv/kyua#16]
_From [jmmv@google.com](https://code.google.com/u/115749396596694607213/) on March 19, 2011 10:54:48_

The main (and probably major) consumer of ATF at this point is the NetBSD test suite.  We must ensure that the test results delivered by Kyua do not deviate from those of ATF.  I am aware that at the problem this is not true, so this issue is intended to fix all the problems in this area.

_Original issue: http://code.google.com/p/kyua/issues/detail?id=16_

**Comment by jmmv** [comment]
_From [jmmv@google.com](https://code.google.com/u/115749396596694607213/) on March 20, 2011 15:20:11_

This issue was updated by revision r95 .

We need to implement integration tests and add this particular case to
them.  The unit tests we currently have are very good know but they will
be brittle during future refactorings.

**Comment by jmmv** [comment]
_From [jmmv@google.com](https://code.google.com/u/115749396596694607213/) on June 07, 2011 07:17:41_

Some integration tests for the cli-side of the tool have been added.  While thinking of adding more integration tests for things like isolation and requirements checking, I realized it is a waste of time to do it now: such things are already very well-tested in their corresponding module-specific tests, and duplicating the tests (in a slower language: sh) will not help much.

I have just run the recent test suite from NetBSD both as root and as a regular user and the results Kyua yields are the same as those produced by atf-run.  Modulo regressions later on, this ticket provides no additional value and can be closed.  We now have all the required functionality implemented!

**Status:** Fixed
